Cabal/Distribution/Types/VersionRange/Internal.hs

```diff
@@ -235,7 +235,7 @@ hyloVersionRange f g = h where h = f . fmap h . g
 instance Pretty VersionRange where
     pretty = fst . cataVersionRange alg
       where
-        alg AnyVersionF                     = (Disp.text "-any", 0 :: Int)
+        alg AnyVersionF                     = (Disp.empty, 0 :: Int)
```

Comment:
This is wrong. The `Dependency` output have to be modified.

See what's done in #6742. That PR is not complete, I run out of time on Friday, so didn't put it up. Sorry for that. I changed the `Dependency` output as I was updating `parser-tests` outputs anyway.